fe/fe-core/src/main/java/com/starrocks/fs/hdfs/HdfsFsManager.java

```diff
@@ -1193,7 +1193,7 @@ public List<FileStatus> listFileMeta(String path, Map<String, String> properties
             throw new UserException("file not found: " + path, e);
         } catch (Exception e) {
             LOG.error("errors while get file status ", e);
-            throw new UserException("listPath failed", e);
+            throw new UserException(e.getMessage(), e);
```

Comment:
Why not give the operation at the same time;
Like `"Fail to get file status because " + e.getMessage()`

Reply:
updated, @imay PTAL